src/qt/bitcoinamountfield.cpp

```diff
@@ -325,6 +325,9 @@ void BitcoinAmountField::unitChanged(int idx)
     // Use description tooltip for current unit for the combobox
     unit->setToolTip(unit->itemData(idx, Qt::ToolTipRole).toString());
 
+    // Update send coins input box's tooltip
+    amount->setToolTip("The amount of " + unit->itemData(idx, Qt::DisplayRole).toString() + " to send");
```

Comment:
Should be translated, see https://doc.qt.io/qt-5/qobject.html#tr.

Reply:
Personally, I don't think it's useful to parametrize the tooltip on the selected unit. This also makes translation harder. Just mention "the selected unit" in text.

Reply:
Doesn't it seem more user friendly to have it reflect the selected unit instead of saying the unit?

I can change it back to what you suggest through just thought it was a nice change 😀

Reply:
I also tend to prefer the constant tooltip, the unit is already right next to the amount field.

Reply:
Even if we don't go with the dynamic tooltip I'm still happy that I learnt how to and for it to work 😀